subprojects/launcher/src/test/groovy/org/gradle/launcher/daemon/configuration/DaemonParametersTest.groovy

```diff
@@ -115,6 +115,54 @@ class DaemonParametersTest extends Specification {
         jvmDefault << [JavaVersion.VERSION_1_8, JavaVersion.VERSION_1_9]
     }
 
+    def "can configure debug port"() {
+        given:
+        parameters.setDebug(true)
+
+        when:
+        parameters.setDebugPort(port)
+
+        then:
+        parameters.effectiveJvmArgs.contains(address)
+
+        where:
+        port || address
```

Comment:
Can you rename this and following tests to use `debugArgument` rather than `address`? I don't think the name fits what it is at the moment.

Reply:
Good idea. Thank you. Done.